lib/structs/events.cpp

```diff
@@ -177,3 +179,4 @@ getMessageType(const json &obj)
 
 }
 }
+#include <iostream>
```

Comment:
Just a small thing I noticed just now. This is probably not what you want :D

Reply:
No clue where it came from